tests/fixtures/falcon_app/app.py

```diff
@@ -0,0 +1,103 @@
+import falcon
+from pathlib import Path
+
+STATIC_IMAGE_PATH = Path(__file__).parent / 'static'
+
+class FalconApp(object):
+    def on_get(self, req, resp):
+        """Handles GET requests on index (/)"""
+        resp.text = """
+    <html>
+        <head>
+            <title>Hello world from Falcon app</title>
+        </head>
+        <body>
+            <h3>Hello world! This is a home page of Falcon app.</h3>
+            <div>
+            <a href='/second_page/'>LINK</a> to second page.
+            </div>
+        </body>
+    </html>"""
```

Comment:
output files,which are generating by us should end with empty line. I supposed it is called EOF or something like that.
```suggestion
    </html>
"""
```